### ISSUE TITLE
socket debug logs clog up debug feed

### ISSUE THREAD

**Issue by erohmensing** [RasaHQ/rasa#4792]
When trying to debug a bot that i'm running over socket, I find it really hard as my logs are clogged up with websocket stuff, which to me doesn't mean anything (maybe the ping pong is somewhat helpful, but not really)
```
019-11-19 14:06:11 DEBUG    websockets.protocol  - undefined - received solicited pong: 86b6b2af
2019-11-19 14:06:21 DEBUG    websockets.protocol  - undefined < Frame(fin=True, opcode=1, data=b'2', rsv1=False, rsv2=False, rsv3=False)
2019-11-19 14:06:21 INFO     engineio.server  - 9b5a104be6f4485fa5133f4d26714f68: Received packet PING data None
2019-11-19 14:06:21 INFO     engineio.server  - 9b5a104be6f4485fa5133f4d26714f68: Sending packet PONG data None
2019-11-19 14:06:21 DEBUG    websockets.protocol  - undefined > Frame(fin=True, opcode=1, data=b'3', rsv1=False, rsv2=False, rsv3=False)
2019-11-19 14:06:31 DEBUG    websockets.protocol  - undefined > Frame(fin=True, opcode=9, data=b'\xa0b\x1cz', rsv1=False, rsv2=False, rsv3=False)
2019-11-19 14:06:31 DEBUG    websockets.protocol  - undefined < Frame(fin=True, opcode=10, data=b'\xa0b\x1cz', rsv1=False, rsv2=False, rsv3=False)
2019-11-19 14:06:31 DEBUG    websockets.protocol  - undefined - received solicited pong: a0621c7a
2019-11-19 14:06:46 DEBUG    websockets.protocol  - undefined < Frame(fin=True, opcode=1, data=b'2', rsv1=False, rsv2=False, rsv3=False)
2019-11-19 14:06:46 INFO     engineio.server  - 9b5a104be6f4485fa5133f4d26714f68: Received packet PING data None
2019-11-19 14:06:46 INFO     engineio.server  - 9b5a104be6f4485fa5133f4d26714f68: Sending packet PONG data None
2019-11-19 14:06:46 DEBUG    websockets.protocol  - undefined > Frame(fin=True, opcode=1, data=b'3', rsv1=False, rsv2=False, rsv3=False)
```

would be nice if we can hide that stuff or make it configurable not to show it.

**Comment by erohmensing** [comment]
This is truly unbearable when we have the socket connector in the html for the docs pages -- multiple debug logs per second from `websockets.protocol` will probably take up a lot of storage with that many logs.

**Comment by erohmensing** [comment]
Oh, I fixed this in #4915. Oops
